src/components/Menu/TopCodeMenu.tsx

```diff
@@ -11,10 +11,12 @@ const TopCodeMenu = ({ languages, versionData }: { languages: string[]; versionD
   const showDefaultLink = pageLanguage !== DEFAULT_LANGUAGE;
   return showCodeMenu ? (
     <div className="fixed right-0 z-10 mt-64 w-full items-end border-b border-mid-grey bg-white md:col-span-4 md:w-4/5 2xl:w-full">
-      <HorizontalMenu variant={HorizontalMenuVariant.end}>
-        <VersionMenu {...versionData} />
-        <LanguageDropdownSelector language={pageLanguage} languages={languages} showDefaultLink={showDefaultLink} />
-      </HorizontalMenu>
+      <div className="w-full 2xl:w-4/5 h-full">
```

Comment:
Why 2xl:w-4/5 after just 2xl:w-full in the containing div?

Reply:
Not sure about this approach at all, very confusing to re-iterate w-full and unclear what h-full is for. Let's check the ticket together to make sure we understand what is requested @aralovelace